lfr/netlistgenerator/v2/gen_strategies/dropxstrategy.py

```diff
@@ -22,183 +23,175 @@ def reduce_mapping_options(self) -> None:
         # input_fignodes = self._fig.get_input_fignodes
         for fignode_id in self._fig.nodes:
             fignode = self._fig.get_fignode(fignode_id)
+
+            # check if construction node
+            if ConstructionNode(fignode.id).is_explictly_mapped:
+                print("this is map")
```

Comment:
What is this supposed to mean ?

Reply:
that's for if it's explicitly mapped or not. I'll comment out the print statement